### PR TITLE
migration/keymanager

### PR DESCRIPTION
**Pull request by rvilarl**
KeyManager!

I have separated the mix of Note and Key under `NoteParts` that we did in music.ts.

### CODE REVIEW

**Comment by rvilarl** [comment]
@0xfe is there any concern with this PR? It was open long ago.

File: src/keymanager.ts [hunk 1]
```diff
@@ -58,7 +68,13 @@ export class KeyManager {
     return this;
   }
 
-  getAccidental(key) {
+  getAccidental(
```
Comment (0xfe) [inline src/keymanager.ts:71]: This formatting feels off. Can you put the function and params in one line?

**Review by 0xfe — COMMENTED** [review]
Sorry, I may have missed this one.

File: src/keymanager.ts (hunk 1 above)
Comment (rvilarl) [reply src/keymanager.ts]: you are right, fixed

**Comment by rvilarl** [comment]
@0xfe ready

**Comment by 0xfe** [comment]
Looks good.

**Pull request merged**